src/DependencyInjection/GoogleMapsExtension.php

```diff
@@ -8,11 +8,15 @@
 
 namespace HeimrichHannot\GoogleMapsBundle\DependencyInjection;
 
+use HeimrichHannot\GoogleMapsBundle\EventListener\ConsentBridgeListener;
+use Hofff\Contao\Consent\Bridge\HofffContaoConsentBridgeBundle;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
+use function class_exists;
```

Comment:
Why is that necessary? In the core a backslash is used (`\class_exists`). In fact we could use the function without any `use` statement at all, no?

Reply:
It's recommend not using global functions to optimize performance (see https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/). You may use the backslash or an explicit `use function` statement. It's a matter of taste what to use.

Reply:
OK, then we should go with the way it's done in the core which is the backslash version.